package.json

```diff
@@ -12,12 +12,11 @@
   },
   "dependencies": {
     "@openzeppelin/contracts": "5.0.0",
-    "@prb/math": "4.0.2"
+    "@prb/math": "github:smol-ninja/prb-math#test/forge-std-v1.8.1"
```

Comment:
Why not install this from my repo using commit `64e4b1`? Or `main`?

BTW, I merged the [PR](https://github.com/PaulRBerg/prb-math/pull/222) now!